### PR TITLE
Fix typo "frp" to "for"

### PR DESCRIPTION
**Pull request by liyuqian**

### CODE REVIEW

**Comment by lanctot** [comment]
Thanks, nice catch

**Comment by liyuqian** [comment]
Thanks, @lanctot! I have an unrelated question: why is "information set" called "information state" in this repo?

**Comment by lanctot** [comment]
Ah yes, because OpenSpiel is partly an attempt to bridge communities. I have many the last 10 years interacting with people who primarily research traditional RL and I found it easier to explain CFR using this terminology. 

In person I usually switch between them depending on who I an talking to :) I think my favorite in this topic is related counterfactual values and q-values, because it lets us compare the algorithms more directly.

File: open_spiel/algorithms/cfr.cc [hunk 1]
```diff
@@ -423,7 +423,7 @@ void CFRSolverBase::GetInfoStatePolicyFromPolicy(
 // - alternating_player: Optionally only update this player.
 // - reach_probabilities: The reach probabilities of this state.
 // - current_player: Either a player or chance_player_.
-// - action_probs: The action probabilities to use frp this state.
```
Comment (goswamig) [inline open_spiel/algorithms/cfr.cc:426]: Testing

**Pull request merged**